mcpi_addons/minecraft.py

```diff
@@ -186,6 +186,18 @@ def __init__(self, connection):
     def clearAll(self):
         """Clear all old events"""
         self.conn.send(b"events.clear")
+        
+    def pollChatPosts(self):
+        s = self.conn.sendReceive(b"events.pollChatPosts").split("\0")
```

Comment:
This should be `events.chat.posts`

Reply:
This causes the API to hang, might have something to do with the use of two dots being used?

Reply:
I figured out how to do this!